test-end-to-end/webapp-cypress/cypress/support/commands.js

```diff
@@ -52,8 +52,17 @@ Cypress.Commands.add('ustadClearDbAndLogin', (username, password) => {
       username,
       password,
     },
-
   })
+
+//https://medium.com/coinmonks/conditional-testing-incypress-tutorial
+
+cy.contains('Existing user').then(($button) => {
+  if ($button.hasClass('active')) {
```

Comment:
This condition is unclear - why does existing user button hasClass active mean the user is already in the login page?